tube-player/README.md

```diff
@@ -37,29 +40,54 @@ This workshop has been set up to provide you with optional content to allow you
 
 The workshop utilizes the latest Uno Platform features, including:
 
-- [C# Markup](xref:Reference.Markup.GettingStarted)
-- [Extensions](xref:Overview.Features)
-  - [MVUX](xref:Overview.Mvux.Overview)
-    - [Pagination](xref:Overview.Mvux.Advanced.Pagination)
-    - [FeedView](xref:Overview.Mvux.FeedView)
-      - NoneTemplate
-      - ErrorTemplate
-    - [Refit](xref:Overview.Http)
-    - [Region navigation](xref:Overview.Navigation)
-    - [Serialization](xref:Overview.Serialization)
+### Environment setup & Inner loop
+
+- [Hot Reload](xref:Uno.Features.HotReload)
+- [Uno.Check](xref:UnoCheck.UsingUnoCheck)
+
+### Cross-platform development
+
+- [Uno.Resizetizer](xref:Uno.Resizetizer.GettingStarted)
+  - [UnoIcon](xref:Uno.Resizetizer.GettingStarted#unoicon)
+  - [UnoSplashScreen](xref:Uno.Resizetizer.GettingStarted#unosplashscreen)
+- [Template Wizard](xref:Uno.GettingStarted.UsingWizard)
+
+### Battle-tested features for apps
+- [Extensions](xref:Uno.Extensions.Overview)
+    - [Hosting environments](xref:Uno.Extensions.Hosting.Overview#hosting-environments)
+    - [Navigation with regions](xref:Uno.Extensions.Navigation.Advanced.Panel)
+    - [Refit endpoints](xref:Uno.Extensions.Http.HowToRefit)
+    - [Serialization](xref:Uno.Extensions.Serialization.Overview)
+- [C# Markup](xref:Uno.Extensions.Markup.Overview)
+
+### Clean architecture, simplified
+
+- [MVUX](xref:Uno.Extensions.Mvux.Overview)
+  - Creating [Feeds](xref:Uno.Extensions.Mvux.Feeds)
+  - Consuming feed data using [FeedView](xref:Uno.Extensions.Mvux.FeedView) with a custom [ErrorTemplate](xref:Uno.Extensions.Mvux.FeedView#errortemplate) and [NoneTemplate](xref:Uno.Extensions.Mvux.FeedView#nonetemplate)
+  - [Paginating](xref:Uno.Extensions.Mvux.Advanced.Pagination) feed data
+
+### Figma
+
 - [Figma plugin](https://aka.platform.uno/uno-figma)
-  - C# Markup
-  - Color overrides
-- [Color overrides](xref:uno.themes.material.getstarted)
-- [DSP import](xref:Uno.Material.DSP)
-- [Material design theme](xref:uno.themes.material.getstarted)
+- [Design to code](xref:Uno.Figma.GetStarted.DesignToCode)
+- [Custom colors](xref:Uno.Figma.Learn.Developers.CustomColors)
+- [Export as C# Markup](xref:Uno.Figma.Learn.Developers.Tabs.Export#c-markup-export)
+
+### Theming
+
+- [Themes](xref:Uno.Themes.Overview)
```

Comment:
@kazo0 in your opinion should we put "Uno.Themes" instead of "Themes" only here ?
(Same for Toolkit, Extensions, "Uno Figma plugin" instead of "Figma plugin"

Reply:
I generally try to use Uno Themes, Uno Extensions, Uno Toolkit. Without the ".", it's still a little mixed in the docs but I'm trying to move it to without the "." as I touch different doc files